lib/Limitation/LimitationValueMapperInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace EzSystems\RepositoryForms\Limitation;
+
+use eZ\Publish\API\Repository\Values\User\Limitation;
+
+/**
+ * Interface for Limitation Value mappers.
+ */
+interface LimitationValueMapperInterface
+{
+    /**
+     * Map limitation value to human readable format.
```

Comment:
Doc could be better here. In some cases an array of strings is returned, that is indeed human readable. But in other cases it's an array of Section objects, or ContentType, or Language, and those are not human readable per se, until they get rendered further down the line.